scripts/lint-style.lean

```diff
@@ -12,10 +13,58 @@ import Cli.Basic
 
 This files defines the `lint-style` executable which runs all text-based style linters.
```

Comment:
```suggestion
This file defines the `lint-style` executable which runs all text-based style linters.
```
this correction probably cannot be successfully applied, since it is outside of the modified lines.